CLAUDE.md

```
@@ -6,19 +6,118 @@ This file provides guidance to Claude Code (claude.ai/code) when working with co
 
 CARE is a Digital Public Good building an open source EMR + Hospital Management system. This is the React frontend (React 19 + TypeScript + Vite).
 
+## Local Development Environment
+
+### Backend Setup (care)
+
+Clone the [care backend](https://github.com/ohcnetwork/care) alongside this repo and create a Python 3.13 venv with dependencies installed.
+
+**Start backend services:**
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)